e2e/cypress/integration/integrations/regen_token_spec.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+let token1;
+let token2;
+let testTeam;
+
+describe('Integrations', () => {
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.visit(`/${team.name}/integrations/commands/add`);
+        });
+    });
+    after(() => {
+        // # clean up - remove slash command
```

Comment:
ah ok. The last one I did that installed the demo plugin required cleanup so I assumed it would be required here. I'll remove.

Reply:
It's case to case basis that's why I also mentioned there that [we're not normally cleaning up state](https://github.com/mattermost/mattermost-webapp/pull/6573#discussion_r497459633). We're testing team/channel/user in isolation by `cy.apiInitSetup`, and slash command is tied up to a team so it's fine not to clean up. Demo plugin is system-wide so it needs to remove after testing since it'll have an effect to other test.